.buildbot.sh

```diff
@@ -1,8 +1,19 @@
 #!/bin/sh
-#
-# Build script for continuous integration.
 
-PATH=/opt/rust/bin:${PATH}
-LLVM_READOBJ_PATH=/opt/llvm-stackmapv3-tools/build/bin/llvm-readobj
+#Cleanup code should always happen
+cleanup() {
+    rm -rf .rustup/
```

Comment:
Are you sure that is working? I think the build doesn't begin in ${HOME}, so it's likely these are a no-op?

```
root@bencher12:/home# find buildbot-worker* -name '.rustup' -type d
buildbot-worker1/.rustup
buildbot-worker2/.rustup
buildbot-worker3/.rustup
```

Reply:
No, you're right. This should be `rm -rf $HOME/.rustup/`

Reply:
Or `~/.rustup`